src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefReadonlyReturnTests.cs

```diff
@@ -28,7 +28,7 @@ class Program
 }
 ";
 
-            var comp = CompileAndVerify(text, parseOptions: TestOptions.Regular);
+            var comp = CompileAndVerify(text, verify: false, parseOptions: TestOptions.Regular);
```

Comment:
Most of the tests for this feature will fail PEVerify with a message similar to `Return type is ByRef, TypedReference, ArgHandle, or ArgIterator.` It is simply not supported yet. I'll chat with Vlad to know if we should partially verify some of the tests (in corner cases PEVerify will not fail), and if I should do this in this PR or not.

---
In reply to: [115337794](https://github.com/dotnet/roslyn/pull/18715#discussion_r115337794) [](ancestors = 115337794)